R/pkg/R/mllib_classification.R

```diff
@@ -111,10 +112,10 @@ setMethod("spark.svmLinear", signature(data = "SparkDataFrame", formula = "formu
             new("LinearSVCModel", jobj = jobj)
           })
 
-#  Predicted values based on an LinearSVCModel model
+#  Predicted values based on a linear SVM model.
```

Comment:
I think these are intentional - we have `#  Predicted values based on an LogisticRegressionModel model`
they are prefix by `#` and not in generated doc - only for developers